parity/deprecated.rs

```diff
@@ -14,6 +14,8 @@
 // You should have received a copy of the GNU General Public License
 // along with Open Ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
+#![allow(dead_code, unused_mut, unused_variables)]
```

Comment:
```suggestion

```

Reply:
`dead_code` is necessary for `Deprecated`. Fixed otherwise.